amida-tech/amida-messaging-microservice
# Feature/support tf deployments

If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
Adds support for Terraform deployments
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/DEVOPS-82
#### How should this be manually tested?
* Be sure to have Packer and Terraform installed on local machine
* Build AMI for deployment with Packer. See https://github.com/amida-tech/amida-messaging-microservice/pull/6. <ami_name> supplied to build command should be `api-messaging-service-boilerplate`
* Enter AWS access key and secret keys in `vars.tf` file
* run `terraform plan` to validate config
* run `terraform apply` to deploy
* To get SNS Alarm notifications be sure that you are subscribed to SNS topic `arn:aws:sns:us-west-2:844297601570:ops_team_alerts` and you have confirmed subscription

#### Any background context you want to provide?
#### Screenshots (if appropriate):
